[CI] Failure during `Install requirements` from `choco` error: `429 (Too Many Requests).`

@geomin12 pointed out this failure with choco during `Install requirements` with this run in `rocm-libraries` repo:
https://github.com/ROCm/rocm-libraries/actions/runs/18408826729/job/52455460205#step:8:26
```haskell
Chocolatey v2.5.1
Installing the following packages:
ccache
By installing, you accept licenses for the packages.
Downloading package from source 'https://community.chocolatey.org/api/v2/'
[NuGet] Error downloading 'ccache.4.12.1' from 'https://community.chocolatey.org/api/v2/package/ccache/4.12.1'.
[NuGet] Response status code does not indicate success: 429 (Too Many Requests).
[NuGet] Error downloading 'ccache.4.12.1' from 'https://community.chocolatey.org/api/v2/package/ccache/4.12.1'.
[NuGet] Response status code does not indicate success: 429 (Too Many Requests).
ccache not installed. An error occurred during installation:
 Error downloading 'ccache.4.12.1' from 'https://community.chocolatey.org/api/v2/package/ccache/4.12.1'.
ccache package files install failed with exit code 1. Performing other installation steps.
The install of ccache was NOT successful.
ccache not installed. An error occurred during installation:
 Error downloading 'ccache.4.12.1' from 'https://community.chocolatey.org/api/v2/package/ccache/4.12.1'.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - ccache (exited 1) - ccache not installed. An error occurred during installation:
 Error downloading 'ccache.4.12.1' from 'https://community.chocolatey.org/api/v2/package/ccache/4.12.1'.
Error: Process completed with exit code 1.
```
This occurred around (10/10/2025 10:37 AM PDT), but it looks like this has been occurring in general across TheRock repo as well (and probably with any repo CI using `choco`):

At around 12:15pm PDT, noticed that 1 of the 2 windows builds successfully finishes `Install requirements` within the same overall CI workflow:

Fails: https://github.com/ROCm/TheRock/actions/runs/18416113588/job/52480223341#step:5:31
Passes: https://github.com/ROCm/TheRock/actions/runs/18416113588/job/52480223321#step:5:107

Complete failures on first package (ccache):
https://github.com/ROCm/TheRock/actions/runs/18415979266/job/52479731291#step:5:31
https://github.com/ROCm/TheRock/actions/runs/18414974176/job/52476469131#step:5:31
https://github.com/ROCm/TheRock/actions/runs/18414675737/job/52475520195#step:5:31
https://github.com/ROCm/TheRock/actions/runs/18414480801/job/52474880713#step:5:32
https://github.com/ROCm/TheRock/actions/runs/18414118034/job/52473578275#step:5:31

Around 10:45 AM PDT, choco fails on the last package, `awscli`, after installing `ccache`, `ninja`, `strawberryperl` successfully:
https://github.com/ROCm/TheRock/actions/runs/18411128999/job/52463466238#step:5:92
```haskell
Chocolatey v2.5.1
Installing the following packages:
strawberryperl
By installing, you accept licenses for the packages.
Downloading package from source 'https://community.chocolatey.org/api/v2/'

strawberryperl v5.42.0.1 [Approved]
strawberryperl package files install completed. Performing other installation steps.
Downloading strawberryperl 64 bit
  from 'https://github.com/StrawberryPerl/Perl-Dist-Strawberry/releases/download/SP_54201_64bit/strawberry-perl-5.42.0.1-64bit.msi'

Download of strawberry-perl-5.42.0.1-64bit.msi (198.4 MB) completed.
Hashes match.
Installing strawberryperl...
strawberryperl has been installed.
  strawberryperl may be able to be automatically uninstalled.
Environment Vars (like PATH) have changed. Close/reopen your shell to
 see the changes (or in powershell/cmd.exe just type `refreshenv`).
 The install of strawberryperl was successful.
  Deployed to 'C:\Strawberry\'

Chocolatey installed 1/1 packages. 
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Chocolatey v2.5.1
Installing the following packages:
awscli
By installing, you accept licenses for the packages.
Downloading package from source 'https://community.chocolatey.org/api/v2/'
[NuGet] Error downloading 'awscli.2.31.11' from 'https://community.chocolatey.org/api/v2/package/awscli/2.31.11'.
[NuGet] Response status code does not indicate success: 429 (Too Many Requests).
[NuGet] Error downloading 'awscli.2.31.11' from 'https://community.chocolatey.org/api/v2/package/awscli/2.31.11'.
[NuGet] Response status code does not indicate success: 429 (Too Many Requests).
awscli not installed. An error occurred during installation:
 Error downloading 'awscli.2.31.11' from 'https://community.chocolatey.org/api/v2/package/awscli/2.31.11'.
awscli package files install failed with exit code 1. Performing other installation steps.
The install of awscli was NOT successful.
awscli not installed. An error occurred during installation:
 Error downloading 'awscli.2.31.11' from 'https://community.chocolatey.org/api/v2/package/awscli/2.31.11'.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
```

Looks like we are being rate limited somehow.

--- a/.github/workflows/build_windows_artifacts.yml
+++ b/.github/workflows/build_windows_artifacts.yml
@@ -65,7 +65,7 @@ jobs:
         # The first two lines removes the default commmunity feed and uses the internal proxy feed
         run: |
           choco source disable -n=chocolatey
-          choco source add -n=internal -s http://10.0.167.96:8081/repository/choco-proxy/ --priority=1
+          choco source add -n=internal -s http://10.0.167.96:8081/repository/choco-group/ --priority=1
           choco install --no-progress -y ccache
           # ninja pinned due to a bug in the 1.13.0 release:
           # https://github.com/ninja-build/ninja/issues/2616

